### PR TITLE
codelingo mock fix

### PR DESCRIPTION
**Pull request by rstorr**

### CODE REVIEW

File: diffparser.go [hunk 1]
```diff
@@ -121,6 +121,7 @@ func lineMode(line string) (*DiffLineMode, error) {
 		m = ADDED
 	case "-":
 		m = REMOVED
+	// mock fix
```
Comment (waigani) [inline diffparser.go:124]: @codelingobot capture

**Review by waigani — COMMENTED** [review]
testing capture

File: diffparser.go (hunk 1 above)
Comment (waigani) [reply diffparser.go:124]: @codelingobot capture

Comment (waigani) [reply diffparser.go:124]: @stagingcodelingobot capture

Comment (waigani) [reply diffparser.go:124]: @staging-codelingo capture

Comment (waigani) [reply diffparser.go:124]: @codelingobot capture

Comment (waigani) [reply diffparser.go:124]: @codelingobot capture

File: diffparser.go [hunk 2]
```diff
@@ -121,6 +121,7 @@ func lineMode(line string) (*DiffLineMode, error) {
 		m = ADDED
 	case "-":
 		m = REMOVED
```
Comment (rstorr) [inline diffparser.go:123]: this is bad

File: diffparser.go [hunk 3]
```diff
@@ -121,6 +121,7 @@ func lineMode(line string) (*DiffLineMode, error) {
 		m = ADDED
 	case "-":
```
Comment (waigani) [inline diffparser.go:122]: Test comment

Comment (waigani) [reply diffparser.go:122]: @codelingobot capture

File: diffparser.go (hunk 1 above)
Comment (waigani) [reply diffparser.go:124]: @codelingobot capture

File: diffparser.go (hunk 3 above)
Comment (waigani) [reply diffparser.go:122]: @codelingobot capture

File: diffparser.go (hunk 1 above)
Comment (rstorr) [reply diffparser.go:124]: @CodeLingoBot capture

File: diffparser.go (hunk 3 above)
Comment (waigani) [reply diffparser.go:122]: @codelingobot capture

Comment (waigani) [reply diffparser.go:122]: @stagingcodelingobot capture

Comment (waigani) [reply diffparser.go:122]: @stagingcodelingobot capture

Comment (waigani) [reply diffparser.go:122]: @codelingobot capture

Comment (rstorr) [reply diffparser.go:122]: @CodeLingoBot capture

Comment (rstorr) [reply diffparser.go:122]: @CodeLingoBot capture

Comment (rstorr) [reply diffparser.go:122]: @CodeLingoBot capture

Comment (waigani) [reply diffparser.go:122]: @CodeLingoBot capture

Comment (waigani) [reply diffparser.go:122]: @codelingobot capture

**Pull request closed without merging**